<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpecificLoadingRulesDialog</class>
 <widget class="QDialog" name="SpecificLoadingRulesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inclusion - Exclusion rules set up</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Patient Loading Rules</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Patient Folder Name Rules</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_8">
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Please, define specific inclusion and exclusion criteria for  the selection fo the data for the patients</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>INCLUDE</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Preffix</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="patientFolderIncludePrefixlineEdit"/>
     </item>
     <item row="0" column="3" rowspan="2">
      <widget class="Line" name="line_7">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>EXCLUDE</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Prefix</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QLineEdit" name="patientFolderExcludePrefixlineEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Suffix</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="patientFolderIncludeSuffixlineEdit"/>
     </item>
     <item row="1" column="5">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Suffix</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QLineEdit" name="patientFolderExcludeSuffixlineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>File Name Rules in a Patient</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>File Extensions for Images</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="1">
      <widget class="QLineEdit" name="patientFileExtensionIncludeSuffixlineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_19">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>INCLUDE</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" rowspan="2">
      <widget class="Line" name="line_9">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLineEdit" name="patientFileExtensionExcludeSuffixlineEdit"/>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_21">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>EXCLUDE</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_10">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_25">
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>If you want to add several extensions, please separate them with commas</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SpecificLoadingRulesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SpecificLoadingRulesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
